src/main/java/org/littil/api/auth/provider/auth0/service/Auth0AuthenticationService.java

```diff
@@ -35,19 +35,10 @@ public class Auth0AuthenticationService implements AuthenticationService {
     private final Auth0UserMapper auth0UserMapper;
     private final Auth0ManagementAPI auth0api;
     private final Auth0RoleService roleService;
-    private final UserService userService;
```

Comment:
agree; based on your comment i cleaned up the use of UserService here @MarcelWildenburg

Reply:
nice work. no comments.

Reply:
PR can be merged with main